timemachine/integrator.py

```diff
@@ -88,6 +88,74 @@ def step(self, x, v):
         return new_x, new_v
 
 
+class VelocityVerletIntegrator(Integrator):
+    def __init__(self, force_fxn, masses, dt):
```

Comment:
A docstring describing the preference of `multiple_steps` over `step` to avoid running 2x force computations in a tight loop would be nice.

Reply:
Added warnings in `__init__` and `step` docstrings about performance https://github.com/proteneer/timemachine/pull/776/commits/4e4a0e1fc20059ff6cc3261cc4bf04325923d1d6